base/src/main/java/org/robojackets/apiary/base/ui/update/UpdateAvailable.kt

```diff
@@ -0,0 +1,109 @@
+package org.robojackets.apiary.base.ui.update
+
+import android.app.Activity
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.google.android.play.core.ktx.AppUpdateResult
+import org.robojackets.apiary.base.ui.icons.UpdateIcon
+import org.robojackets.apiary.base.ui.util.getActivity
+import se.warting.inappupdate.compose.rememberInAppUpdateState
+import timber.log.Timber
+
+// Just a random number so we can identify our update request later if necessary
+const val UPDATE_REQUEST_CODE = 1999
```

Comment:
What exactly is this for?

Reply:
When you call the start update function, I believe you have to provide it as an identifier so you can get the status of your update request later. I think(?) this is used somewhere just a few lines below

Reply:
Weird API but ok